rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -342,9 +390,155 @@ impl CalculatorBase for LodeSphericalExpansion {
             let k_vectors = compute_k_vectors(&cell, k_cutoff);
             assert!(!k_vectors.is_empty());
 
-            self.project_k_to_nlm(&k_vectors, do_gradients);
+            self.project_k_to_nlm(&k_vectors);
+
+            let structure_factors = compute_structure_factors(
+                system.positions()?,
+                system.species()?,
+                &k_vectors
+            );
+
+            let density_fourrier = self.compute_density_fourrier(&k_vectors);
+
+            let global_factor = 4.0 * std::f64::consts::PI / cell.volume();
```

Comment:
I am not happy with the name `global_factor`. Maybe we find something more descriptive. A bit better would be to call it `normalization`.

Reply:
That works, this is just how it is also called in the SOAP spherical expansion. But we can change both!